src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationAnnArrInit2.java

```diff
@@ -0,0 +1,53 @@
+/* Config:                                                                  //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration: //indent:1 exp:1
+ *                                                                          //indent:1 exp:1
+ * arrayInitIndent = 0                                                      //indent:1 exp:1
+ * basicOffset = 2                                                          //indent:1 exp:1
+ * braceAdjustment = 0                                                      //indent:1 exp:1
+ * caseIndent = 4                                                           //indent:1 exp:1
+ * forceStrictCondition = false                                             //indent:1 exp:1
+ * lineWrappingIndentation = 0                                              //indent:1 exp:1
+ * tabWidth = 8                                                             //indent:1 exp:1
+ * throwsIndent = 4                                                         //indent:1 exp:1
+ */                                                                         //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+@InputIndentationAnnArrInit2.Foo({ //indent:0 exp:0
+    @InputIndentationAnnArrInit2.Bar, //indent:4 exp:>=0
```

Comment:
Continuation of https://github.com/checkstyle/checkstyle/pull/8083#discussion_r530751111 ,

Should 0 really be allowed as an indentation? 
Even google style examples show some indentation in the example provided.
https://google.github.io/styleguide/javaguide.html#s4.8.3.1-array-initializers

Reply:
@rnveach 
Actually it is checking against config where arrayInit = 0, lineWrapping = 0. 
and thus allowing 0 as an indentation level.

Since google has 2 as arrayInit, and 4 as lineWrapping. It will not allow the same.

Reply:
Thanks, sorry, I missed the config.

Reply:
@Abhishek-kumar09 , 
`0` is not good indentation.
please make code sample like this with `indent:0` and should violate such code.